src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -144,11 +144,10 @@ function getDrupalContent(buildOptions) {
       loadCachedDrupalFiles(buildOptions, files);
       pipeDrupalPagesIntoMetalsmith(drupalData, files);
       log('Successfully piped Drupal content into Metalsmith!');
-
-      metalsmith.metadata({ drupalData });
+      buildOptions.drupalData = drupalData;
       done();
     } catch (err) {
-      metalsmith.metadata({ drupalError: drupalData });
+      buildOptions.drupalError = drupalData;
```

Comment:
😫

Reply:
Ugh, I really thought that `metalsmith.metadata()` worked like React's `setState`, where it would just add object properties together instead of just a complete replacement.